bin/resource_management/resource-broken-links-csv.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import pandas as pd
```

Comment:
pandas isn't the best tool for processing these jsonl/csv files. Pandas loads the complete file into memory instead of streaming it. It's better to use standard python csv/json modules. Streaming lines also lets you divide work evenly across the workers instead of breaking into groups at the beginning and hoping that all groups complete at about the same time.